package.json

```diff
@@ -9,6 +9,7 @@
   },
   "dependencies": {
     "@quasar/extras": "^1.12.5",
+    "css": "^3.0.0",
```

Comment:
why are we including css as an npm package? I didn't see it used anywhere. Most CSS stuff will be in scoped style tags, which I see you figured out mostly below.